torchci/components/benchmark/llms/SummaryPanel.tsx

```diff
@@ -27,6 +28,7 @@ export function SummaryPanel({
   startTime: dayjs.Dayjs;
   stopTime: dayjs.Dayjs;
   granularity: Granularity;
+  repoName: string;
   modelName: string;
   metricNames: string[];
   lPerfData: BranchAndCommitPerfData;
```

Comment:
Can we display the same metric format for both Apple and Android? I notice that all data from Apple are integers and Android are float arlunded to 5-6 decimal places.

Reply:
Oh, I think you need to switch to ClickHouse where the queries are correct:

<img width="348" alt="Screenshot 2024-10-14 at 10 57 34" src="https://github.com/user-attachments/assets/bf254f19-a599-48b4-999a-cfd7a6ac00ce">

I will do a follow up PR right after this one to clean up the other option (Rockset) as it will go away soon (by end of this month) and it's a waste of effort to keep the two in sync.

Reply:
Yeah, I switched to CH but still seeing different data formats. Yeah, it's okay to follow up in a separate PR if we still see the issue after this one merged on main

Reply:
Oh, I get what you mean.  This looks like an upstream issue from ExecuTorch apple-perf job though as that's where the data comes from.  The dashboard just shows what it's there.  Let me look into that.

Reply:
Peak memory usages are reported by the iOS app. Can we include that metric in the dashboard? Hansong is experimental that feature for Android in parallel.

Reply:
Yup, that can be done from ExecuTorch side, as long as the metric is uploaded, it will show up automatically on the dashboard